src/main/webapp/index.html

```diff
@@ -20,15 +21,18 @@
       <div id="form-title">
         <p>Choose Type of Input</p>
       </div>
+      <label><input type="checkbox" onclick="setLocationCookie()" /> Use Location</label>
```

Comment:
The PR I merged this morning doesn't have an onclick attribute because there's a listener set up with jQuery in index.js